src/controllers/heroPostController.js

```diff
@@ -0,0 +1,20 @@
+const { addHero } = require('../services/herolist');
+
+const heroPostController = (req, res) => {
+  const { userId, heroName } = { userId: Number(req.headers.userid), heroName: req.headers.name };
```

Comment:
The heroname should not come from the header information, it should come in request body.

Reply:
Fixed, thank you!